Tests/TuistCoreTests/MetadataProviders/XCFrameworkSignatureProviderTests.swift

```diff
@@ -0,0 +1,221 @@
+import FileSystem
+import Mockable
+import Path
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistCore
+
+final class XCFrameworkSignatureProviderTests: TuistUnitTestCase {
```

Comment:
As mentioned elsewhere, I'd consolidate `XCFrameworkSignatureProviderIntegrationTests` into `XCFrameworkSignatureProviderTests` and only add unit tests for the pieces that can't be tested via integration tests.